pkg/interop/convert/convert.go

```diff
@@ -1,6 +1,11 @@
 // Package convert provides functions for type conversion.
 package convert
 
+const (
+	// uintSizeInBytes is the size of a uint in bytes.
+	uintSizeInBytes = (32 << (^uint(0) >> 63)) / 8
+)
```

Comment:
Adjust PR/commit title. It's not `encoding` anymore; these functions are fised-size integer convertors; s/be/BE.